client/lib/post-metadata/index.js

```diff
@@ -136,6 +136,41 @@ PostMetadata = {
 			return [ latitude, longitude ];
 		}
 	},
+
+	/**
+	 * Given a post object, return either "public" or "private", indicating whether
+	 * the geo-location data associated with the post is allowed to be displayed
+	 * publicly.
+	 *
+	 * @param {Object} post Post object
+	 * @returns {string}	Either "public" or "private"
+	 */
+	geoIsSharedPublicly: function( post ) {
+		if ( ! post ) {
+			return 'public';
```

Comment:
What if we returned a boolean true or false instead of `public` and `private`? That's what function that ask an `is` question usually return.

Reply:
Fair enough.  I think early on I wasn't sure if I was going to end up with additional states other than just public/private, but with just those two a straight-up boolean does simplify things.  
 This is done in 07480a0 Thanks!